packages/imperative/src/imperative/src/plugins/utilities/NpmFunctions.ts

```diff
@@ -43,9 +43,12 @@ export function findNpmOnPath(): string {
  * @return {string} command response
  *
  */
-export function installPackages(npmPackage: string, npmArgs: INpmInstallArgs): string {
+export function installPackages(npmPackage: string, npmArgs: INpmInstallArgs, verbose = false): string {
     const pipe: StdioOptions = ["pipe", "pipe", "pipe"];
     const args = ["install", npmPackage, "-g", "--legacy-peer-deps"];
+    if (verbose) {
+        args.push("--loglevel=debug", "--foreground-scripts=true");
```

Comment:
For logLever...
- Should/Can we look at the app log level ENV var to determine how much output we should print?
- Should we also pass the `--loglevel=silly` flag if the user has specified `ZOWE_APP_LOG_LEVEL=TRACE`?
- Generally speaking, should we try to match our app_log_level to NPM's log level?
https://docs.npmjs.com/cli/v8/using-npm/logging#loglevel

That way people can control how much logs are printed out (when they pass the verbose flag)

Reply:
> Should we also pass the `--loglevel=silly` flag if the user has specified `ZOWE_APP_LOG_LEVEL=TRACE`?

I like this idea, and it seems like a good enough option for short-term - implemented in https://github.com/zowe/zowe-cli/pull/2562/commits/c39d19c05df956e8c35e6c685ed6db9fd0c81b7d

Matching our log level with npm seems outside the scope of this PR and could be addressed in https://github.com/zowe/zowe-cli/issues/2560.